README.md

```diff
@@ -470,9 +470,17 @@ These definitions have objects analogous to an `OpenLDAP` entry definition and s
                 LdapObjectDefinition.builder("TestUser")
                         .setDistinguishedName("uid=testuser,ou=Test,dc=tempto,dc=com")
                         .setAttributes(testUserAttributes())
+                        .addModificationAttributes(memberOfAttributes())
                         .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
                                 .build();
 
+        private static Map<String, List<String>> memberOfAttributes()
+        {
+                Map<String, List<String>> attributes = newHashMap();
```

Comment:
As a rule of thumb if `Map<>` is returned we should use `ImmutableMap` - not a HashMap.